src/General-Rules/ReSuperSendsRule.class.st

```diff
@@ -40,3 +33,9 @@ ReSuperSendsRule >> initialize [
 ReSuperSendsRule >> name [
 	^ 'Rewrite super messages to self messages'
 ]
+
+{ #category : 'testing' }
+ReSuperSendsRule >> shouldCheckMethod: aMethod [
+
+	^ aMethod methodClass withAllSubclasses noneSatisfy: [ :class | class includesSelector: aMethod selector ]
```

Comment:
why this is not aMethod methodClass class withAllSuclasses?

Reply:
Argh this is another rule.

Reply:
I just to not get this code :)

Reply:
It’s another rule and since I optimized the first one I also did this one